src/main/java/mongodb/Main.java

```diff
@@ -0,0 +1,220 @@
+    } catch (MongoException me) {
+      System.err.println("Unable to insert any driver documents due to an error: " + me);
+    }
+
+    /*      *** FIND DOCUMENTS ***
+     *
+     * Now that we have data in Atlas, we can read it. To retrieve all of
+     * the data in a collection, we call find() with an empty filter.
+     */
+
+    try (MongoCursor<Recipe> cursor = collection.find().iterator()) {
+      while (cursor.hasNext()) {
+        System.out.println(cursor.next());
+      }
+    } catch (MongoException me) {
+      System.err.println("Could not find documents due to an error: " + me);
+    }
+
+    // We can also find a single document. Let's find the first document
+    // that has the string "potato" in the Ingredients list. We
+    // use the Filters.eq() method to search for any values in any
+    // ingredients list that match the string "potato":
+
+    Bson findPotato = eq("ingredients", "potato");
+    try {
+      Recipe firstPotato = collection.find(findPotato).first();
+      if (firstPotato == null) {
+        System.out.println("Didn't find any recipes containing 'potato' as an ingredient.");
+      } else {
+        System.out.println(firstPotato);
+      }
+    } catch (MongoException me) {
+      System.err.println("Could not find single document due to an error: " + me);
+    }
+
+    /*      *** UPDATE A DOCUMENT ***
+     *
+     * You can update a single document or multiple documents in a single call.
+     *
+     * Here we update the PrepTimeInMinutes value on the document we
+     * just found.
+     */
+    Bson updateFilter = Updates.set("prepTimeInMinutes", 72);
+
+    // The following FindOneAndUpdateOptions specify that we want it to return
+    // the *updated* document to us. By default, we get the document as it was *before*
+    // the update.
+    FindOneAndUpdateOptions options = new FindOneAndUpdateOptions();
+    options.returnDocument(ReturnDocument.AFTER);
+
+    // The updatedDocument object is a Recipe object that reflects the
+    // changes we just made.
+    try {
+      Recipe updatedDocument = collection.findOneAndUpdate(findPotato,
+              updateFilter, options);
+      if (updatedDocument == null) {
+        System.out.println("Couldn't update the recipe.");
+      } else {
+        System.out.println("Updated the recipe to: " + updatedDocument);
+      }
+    } catch (MongoException me) {
+      System.err.println("Could not update any recipes due to an error: " + me);
+    }
+
+    /*      *** DELETE DOCUMENTS ***
+     *
+     *      As with other CRUD methods, you can delete a single document
+     *      or all documents that match a specified filter. To delete all
+     *      of the documents in a collection, pass an empty filter to
+     *      the deleteMany() method. In this example, we'll delete 2 of
+     *      the recipes.
+     */
+    Bson deleteFilter = Filters.in("name", Arrays.asList("elotes", "fried rice"));
```

Comment:
For clarity, I wonder if it may be better to stick to a single way of referring to the `Filters` methods. Prior to this, you used `eq()` to call a static method in Filters, but here you're using `Filters.in()` rather than `in()`.